# This message provides available links and names with a plugin and also few other useful info
string[] link_names #LinkNames (These are also available in a plugin and are repeated only for convenience)
string[] joint_names #JointNames
bool[] joint_types #Joint Types
